Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.cs

```diff
@@ -184,7 +184,7 @@ protected override Size MeasureOverride(Size availableSize)
             }
 
             // update value with the last line
-            // if the the last loop is(parentMeasure.U > currentMeasure.U + lineMeasure.U) the total isn't calculated then calculate it
+            // if the last loop is(parentMeasure.U > currentMeasure.U + lineMeasure.U) the total isn't calculated then calculate it
```

Comment:
Should we also take this occasion to fix the spacing here?
```suggestion
            // if the last loop is (parentMeasure.U > currentMeasure.U + lineMeasure.U) the total isn't calculated then calculate it
```

Reply:
nice catch